src/markdown-pages/build-apps/serve-publish-subscribe/catalog.mdx

```diff
@@ -141,6 +141,10 @@ ls nerdlets/home/catalog
 [output] {blue}screenshots
 ```
 
+After adding a screenshot into each directory, click your nerdpack to see the changes.
+
+![What's Inside Tab](../../../images/build-an-app/whats-inside-nerdlet-tab.png)
```

Comment:
Do you think this will be a bit confusing to jump to the Account Maturity app when the other screenshots in this guide are referencing the Demo App?

I think we'll need to recreate this process for that Demo App for the sake of consistency